src/games/rota/game.ts

```diff
@@ -0,0 +1,166 @@
+import { IGameArgs, IGameCtx, INVALID_MOVE } from 'boardgame.io/core';
+
+// players are either allowed to place or move pieces
+export enum Phase {
+  Place = 'Place',
+  Move = 'Move',
+}
+
+export interface Point {
+  x: number;
+  y: number;
+  playerID: string;
+  pieceID: number;
+}
+
+export interface IG {
+  points: Point[];
+  piecesPlaced: number;
+  selectedPoint: Point;
+}
+
+function findPoint(G: IG, pointX: number, pointY: number): Point {
+  for (const p of G.points) {
+    if (p.x === pointX && p.y === pointY) {
+      return p;
+    }
+  }
+  return { x: null, y: null, playerID: null, pieceID:null };
+}
+
+function isVictory(G: IG): boolean {
+  for (const p of G.points) {
+    if (p.playerID !== null) {
+      const axisVar: number[] = [-1, 0, 1];
+      // check along x-axis
+      let testWin = true;
+      for (const y of axisVar) {
+        testWin = testWin && p.playerID === findPoint(G, p.x, y).playerID;
+      }
+      if (testWin) {
```

Comment:
move this test to its own function... A good rule of thumb is that whenever you feel like commenting a sub part of a function, it is probably a good idea to create a function for that subpart